pyproject.toml

```diff
@@ -23,6 +23,52 @@ setup_env = {cmd = 'nox -s setup --', help = 'Setup the local environment and se
 test = { cmd = "nox -Rs test --", help = "Run pytest" }
 
 
+[tool.towncrier]
+template = 'changelog/_template.rst'
+package = "disnake"
+filename = "docs/whats_new.rst"
+directory = "changelog/"
+title_format = "v{version}"
+underlines = "-~"
+issue_format = ":issue:`{issue}`"
+
+    [[tool.towncrier.type]]
+    directory = 'breaking'
```

Comment:
small nit, these should all use double quotes to be consistent with the rest of the config file

Reply:
single and double quotes in toml have a different meaning. We should probably standardize when possible, but single quotes are equal to the `r` string prefix in python.

Reply:
https://github.com/DisnakeDev/disnake/pull/600/commits/d813f5ad596a02b3cf174827f73b951d9f5dd6fd